diff -urNp -x '*.orig' compat-ncurses5-5.9.20150117.org/misc/terminfo.src compat-ncurses5-5.9.20150117/misc/terminfo.src
--- compat-ncurses5-5.9.20150117.org/misc/terminfo.src	2023-07-20 23:59:03.823541888 +0200
+++ compat-ncurses5-5.9.20150117/misc/terminfo.src	2023-07-20 23:59:04.799555886 +0200
@@ -3607,7 +3607,7 @@ putty-sco|putty with SCO function keys,
 # "resize -s"), though it does not pass SIGWINCH to the application if the
 # user resizes the window with the mouse.
 teraterm2.3|Tera Term Pro,
-	km, xon@,
+	xon@,
 	ncv#43, vt@,
 	acsc=+\020\,\021-\030.^Y0\333`\004a\261f\370g\361h\260i\316j\331k\277l\332m\300n\305o~p\304q\304r\304s_t\303u\264v\301w\302x\263y\363z\362{\343|\330}\234~\376,
 	blink=\E[5m, bold=\E[1m, civis=\E[?25l, clear=\E[H\E[J,
@@ -3786,7 +3786,7 @@ xterm-r5|xterm R5 version,
 # (khome/kend do not actually work in X11R5 or X11R6, but many people use this
 # for compatibility with other emulators).
 xterm-r6|xterm X11R6 version,
-	OTbs, am, km, mir, msgr, xenl,
+	OTbs, am, mir, msgr, xenl, G0,
 	cols#80, it#8, lines#24,
 	acsc=``aaffggiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~,
 	bel=^G, bold=\E[1m, clear=\E[H\E[2J, cr=^M,
@@ -5891,7 +5891,7 @@ eterm-color|Emacs term.el terminal emula
 # sgr are not impacted (because that usually resets all capabilities before
 # setting any), but termcap applications do not use sgr -TD
 screen|VT 100/ANSI X3.64 virtual terminal,
-	OTbs, OTpt, am, km, mir, msgr, xenl, G0,
+	OTbs, OTpt, am, mir, msgr, xenl, G0,
 	colors#8, cols#80, it#8, lines#24, ncv@, pairs#64, U8#1,
 	acsc=++\,\,--..00``aaffgghhiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~,
 	bel=^G, blink=\E[5m, bold=\E[1m, cbt=\E[Z, civis=\E[?25l,
